application/features/bootstrap/EmailContext.php

```diff
@@ -228,6 +231,52 @@ public function thatUserDoesNotHaveAPassword()
         }
     }
 
+    /**
+     * @Given a :messageType email was sent :daysAgo days ago to that user
+     */
+    public function anEmailWasSentXDaysAgoToThatUser(string $messageType, int $daysAgo)
+    {
+        if ($daysAgo < 0) {
+            return;
+        }
+
+        $emailLog = new EmailLog([
+            'user_id' => $this->tempUser->id,
+            'message_type' => $messageType,
+        ]);
+
+        if ($daysAgo > 0) {
+            $diffConfig = "-" . $daysAgo . " days";
+            $dbDate = MySqlDateTime::relative($diffConfig);
+
+            $emailLog->sent_utc = $dbDate;
+        }
+        echo PHP_EOL . "Date: " . $emailLog->sent_utc;
+        $emailLog->save();
+    }
+
+    /**
+     * @Given a :messageType email has been sent to that user
+     */
+    public function anEmailHasBeenSentToThatUser(string $messageType)
+    {
+
+        $emailLog = new EmailLog([
+            'user_id' => $this->tempUser->id,
+            'message_type' => $messageType,
+        ]);
+
+        $emailLog->save();
+    }
+
+    /**
+     * @Given a :messageType email has NOT been sent to that user
+     */
+    public function anEmailHasNotBeenSentToThatUser(string $messageType)
+    {
+        return;
```

Comment:
Looks like the body of this function was never written. I think all that it would need to do is delete any emails of that type that were sent to that user.